R/doeFactorial.R

```diff
@@ -92,27 +92,36 @@ doeFactorial <- function(jaspResults, dataset, options, ...){
     designs <- jaspResults[["state"]]$object
 
     if(options[["designBy"]] == "designByRuns"){
-      rowIndex 	    <- which(designs[, 1] == options[["factorialRuns"]])
-      resolution    <- designs[rowIndex, options[["numberOfFactors"]]]
-      runs          <- options[["factorialRuns"]]
+      runs <- options[["factorialRuns"]]
+      if(log2(as.numeric(options[["factorialRuns"]])) < options[["numberOfFactors"]]){
+      resolution <- DoE.base::design.info(FrF2::FrF2(nfactors = as.numeric(options[["numberOfFactors"]]),
```

Comment:
if you add a package you should also add it to the DESCRIPTION file.

Reply:
ah okay, I thought that since DoE.base is a dependency of FrF2, I didn't have to add it, will do so now

Reply:
in that case, it would have probably worked. But that's not how it's supposed to be.